### ISSUE TITLE
Loss is high 

### ISSUE THREAD

**Issue by Dhirendramohanjha** [vijayg15/FaceMask-Detection#4]
Hi my data txt file looks like 
/YOLOv3/image/20398.jpg 1,0,63,155,1 92,19,169,101,1 198,45,300,122,1 
/YOLOv3/image/20408.jpg 148,36,207,106,1 78,33,131,96,1 240,0,279,60,0 
/YOLOv3/image/20283.jpg 337,214,481,394,1 672,227,834,403,1 992,286,1111,430,1 1127,279,1285,475,1 

after 150 epoch I get training loss 15 and validation loss is 16 approx, so how I minimize these losses.
